cpp/include/cudf/io/detail/json.hpp

```diff
@@ -12,12 +12,7 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- */
-
-/**
- * @file json.hpp
- * @brief cuDF-IO reader classes API
- */
+ */\
```

Comment:
This extra backslash seems like a typo?
```suggestion
 */
```